crates/bitwarden-auth/tests/send_access.rs

```diff
@@ -0,0 +1,532 @@
+//! Integration tests for send access feature
+
+use bitwarden_auth::{
+    send_access::{
+        api::{
+            SendAccessTokenApiErrorResponse, SendAccessTokenInvalidGrantError,
+            SendAccessTokenInvalidRequestError,
+        },
+        SendAccessClient, SendAccessCredentials, SendAccessTokenError, SendAccessTokenRequest,
+        SendAccessTokenResponse, SendEmailCredentials, SendPasswordCredentials,
+    },
+    AuthClientExt,
+};
+use bitwarden_core::{Client as CoreClient, ClientSettings, DeviceType};
+use wiremock::{
+    matchers::{self, body_string_contains},
+    Mock, MockServer, ResponseTemplate,
+};
+
+fn make_send_client(mock_server: &MockServer) -> SendAccessClient {
```

Comment:
You might be able to reuse the `start_api_mock` function in the `bitwarden-test` crate instead of creating the mock server yourself.

https://github.com/bitwarden/sdk-internal/blob/2093c1a015e024b6591c13a1aa2057a7cb0d02d8/crates/bitwarden-test/src/api.rs#L6

Reply:
That's a great helper that I had no clue about. I've updated my tests to use it! Thank you!
c2164adad12089f4dea85b1266c975e2e98c1d90